Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -124,10 +126,28 @@ def tests
           parallel = false
           ["test/#{test_name}_spec.rb:#{line}"]
         end
+      elsif args.changed?
+        changed_files = Utils.safe_popen_read("git", "diff", "--name-only", "master")
+
+        filestub_regex = %r{Library/Homebrew/([A-Za-z/_-]+).rb}
+
+        changed_files.scan(filestub_regex)
+                     .map { |filestub| "test/#{filestub.last}_spec.rb" }
+                     .select { |filename| File.exist?(filename) }
```

Comment:
```suggestion
                     .map { |filestub| Pathname("test/#{filestub.last}_spec.rb") }
                     .select(&:exist?)
```

Reply:
That's a Ruby pattern I didn't know about.